tests/backends/test_mixins.py

```diff
@@ -59,6 +59,7 @@ def test_backends_mixins_history_mixin_write_history(fs, monkeypatch):
     # Looks like pyfakefs needs some help with pathlib overrides (we are using
     # Path().parent in our implementation).
     monkeypatch.setattr("ralph.backends.mixins.settings", Settings())
+    fs.create_dir(str(settings.APP_DIR))
```

Comment:
The comment above doesn't seems related to the monkey-patching. There is a :wolf: here! (:sweat_smile:)

Reply:
Well spotted) Indeed, it might be useful to add to the comment that the `Settings` object has `Path` fields (ex. `APP_DIR`) which needs to be redefined in the scope of this test for `pyfakefs` to override them. Do we create a separate pull request for this?

Reply:
You can handle this in this PR but in a separated commit?

Reply:
Ok) Will be done)

Reply:
Indeed, in the file history, the `APP_DIR` creation was forced and the above comment was related to this

Reply:
Sorry, I was wrong, it looks like the `monkeypatch` was not needed after all.